DEVELOPING.md

```diff
@@ -285,6 +285,11 @@ Possible vulnerability schemas supported within the vunnel repo are:
 - [GitHub Security Advisories](https://github.com/anchore/vunnel/tree/main/schema/vulnerability/github-security-advisory)
 - [NVD Vulnerability](https://github.com/anchore/vunnel/tree/main/schema/vulnerability/nvd)
 
+If at any point a breaking change needs to be made to a provider (and say the schema remains the same), then you
+can set the `__version__` attribute on the provider class to a new integer value (incrementing from `1` onwards). This
```

Comment:
Do you just want these to be 1, 2, 3, 4?

If there is no breaking change does it matter to keep track of additions to the structure?